examples/dwarfdump.rs

```diff
@@ -602,32 +592,40 @@ where
         load_file_section(id, file, endian, flags.dwo, &arena_data, &arena_relocations)
     };
     let mut dwarf = gimli::Dwarf::load(&mut load_section)?;
-    let mut load_dwo_parent_section = None;
+    let mut dwo_parent_units = HashMap::new();
     if flags.dwo {
         dwarf.file_type = gimli::DwarfFileType::Dwo;
 
         if let Some(dwo_parent_file) = flags.dwo_parent.as_ref() {
-            let arena_data = &arena_data;
-            let arena_relocations = &arena_relocations;
-            load_dwo_parent_section = Some(move |id: gimli::SectionId| -> Result<_> {
+            let mut load_dwo_parent_section = |id: gimli::SectionId| -> Result<_> {
                 load_file_section(
                     id,
                     dwo_parent_file,
                     endian,
                     false,
-                    arena_data,
-                    arena_relocations,
+                    &arena_data,
+                    &arena_relocations,
                 )
-            });
-            dwarf.debug_addr = gimli::Section::load(load_dwo_parent_section.as_mut().unwrap())?;
-            dwarf.ranges = gimli::RangeLists::new(
-                gimli::Section::load(load_dwo_parent_section.as_mut().unwrap())?,
-                gimli::Section::load(load_dwo_parent_section.as_mut().unwrap())?,
-            );
+            };
+            let dwo_parent = gimli::Dwarf::load(&mut load_dwo_parent_section)?;
+            dwarf.debug_addr = dwo_parent.debug_addr.clone();
+            dwarf.ranges = dwo_parent.ranges.clone();
```

Comment:
debug_line/debug_line_str really should be copied from the parent, and ranges should be copied if and only if the dwo doesn't have debug_rnglists (in other words if this is the GNU split dwarf extension and not the standard DWARF 5 version)

Reply:
Ok. This isn't something I changed in this PR is it? This sounds like something to be addressed as part of adding a method to `Dwarf` to do this.

Reply:
Right, nothing changed here afaict.